LaTeX/LaTeX.sublime-syntax

```diff
@@ -136,8 +140,12 @@ contexts:
         5: punctuation.definition.group.brace.end.latex
         6: support.function.latex entity.name.newcommand.latex
         7: punctuation.definition.group.bracket.begin.latex
-        8: punctuation.definition.group.bracket.end.latex
-        9: punctuation.definition.group.brace.begin.latex
+        8: constant.numeric.value.latex
```

Comment:
Maybe not high prio in LaTeX, but numbers adding a `meta.number` scope to comply with other syntaxes of this repo would be appreciated.

```suggestion
        8: meta.number.integer.decimal.latex constant.numeric.value.latex
```